doc/user_guide/built_ins.rst

```diff
@@ -17,7 +17,10 @@ specification of what is required without an implementation. Therefore
 the PSy layer is free to implement these operations in whatever way it
 chooses.
 
-.. note:: In general, PSyclone will need to know the types of the arguments being passed to any Built-ins. The parser obtains this information from an API-specific file that contains the metadata for all Built-in operations supported for that API.
+.. note:: In general, PSyclone will need to know the types of the arguments
```

Comment:
As you've changed this file could you please add copyright info and authorship to the top. The dynamo0p3.rst file already has it so you can use that as a template.

Reply:
Yes, I realised I should have done that pretty much as I sent the branch to review. Added the copyright and autorship info as requested.